packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -196,7 +199,7 @@ export default class QuanticResultQuickview extends LightningElement {
 
   dispatchHasPreview(hasPreview) {
     this.dispatchEvent(
-      new CustomEvent('haspreview', {
+      new CustomEvent('quantic__haspreview', {
```

Comment:
This was not updated in the component that listens to this event:

https://github.com/coveo/ui-kit/blob/a6dd19c883664e09a86dd80c81c55e637315ac8e/packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js#L68